web/server/entrypoint.go

```diff
@@ -40,6 +40,7 @@ type Arguments struct {
 	RevealSensitiveConfigDiffs bool   `flag:"reveal-sensitive-config-diffs,usage=show config diffs"`
 	UntrustedEnv               bool   `flag:"untrusted-env,usage=disable processes and shell from running in a untrusted environment"`
 	OutputTelemetry            bool   `flag:"output-telemetry,usage=print out telemetry data (metrics and spans)"`
+	DisableMulticastDNS        bool   `flag:"disable-mdns,usage=disable server discovery through multicast DNS"`
```

Comment:
should this be a "positive" option with a default of `true` like the corresponding `-webrtc` flag? personally, I would prefer to change the webrtc flag to be negative (e.g. `-disable-webrtc`) so the default would be false, but am hesitant to change the existing interface.

thoughts?

Reply:
I think the two should be consistent, and I don't think we should change the existing interface, so I prefer the positive option with a default of `true`

Reply:
I left the option on the underlying struct negative since we can't change the default value on that - we get test failures otherwise: https://github.com/viamrobotics/rdk/pull/2826/commits/6b7c51fc044b444ef7c4382a04ced09289139012

Reply:
anyway to fix the tests? I don't think the two should be different

Reply:
discussed IRL, we'll gonna make the new option negative